<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updateAccount</class>
 <widget class="QWidget" name="updateAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_email">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Email: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_accountType">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Account Type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_firstName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>First Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_lastName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_phoneNumber">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Phone Number:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_address">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Address:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_register">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>370</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Register to a location (enter location key)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_locationKey">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Location Key:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_firstName">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_lastName">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_phone">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_address">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_register">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>400</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_register">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>440</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_update">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>300</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Update Account</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
